src/app/shared/NotificationItem.svelte

```diff
@@ -25,20 +25,11 @@
   </div>
 {/if}
 
-<NoteReducer {events} shouldSort depth={1} bind:items let:event let:getContext>
-  <div class="flex items-center justify-between">
-    {#if getContext(event).length === 0}
-      <PeopleAction pubkeys={[event.pubkey]} actionText="mentioned you" />
-    {:else if event.pubkey === $pubkey}
-      <PeopleAction
-        pubkeys={uniq(pluck("pubkey", getContext(event)))}
-        actionText="{verb} to your note" />
-    {:else}
-      <PeopleAction
-        pubkeys={uniq(pluck("pubkey", getContext(event)))}
-        actionText="{verb} to a note mentioning you" />
-    {/if}
-    <small>{formatTimestamp(max(pluck("created_at", [event, ...getContext(event)])))}</small>
-  </div>
+<NoteReducer {events} depth={1} bind:items let:event let:getContext>
+  {#if kind === "reactions"}
+    <NoteReactions context={getContext(event)} {event} />
+  {:else}
+    <NoteInteractions context={getContext(event)} {event} />
+  {/if}
```

Comment:
The kind parameter has become a little goofy, let's just make this a slot instead and pass in NoteReducer and the correct child component directly from the caller.